src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const COPY_STATE = { ...state };
```

Comment:
The variable 'COPY_STATE' should be declared inside the for loop to ensure it's a fresh copy of the state for each action.